index.md

```diff
@@ -63,7 +63,7 @@ locations:
   The course is aimed at everyone who is
   interested in becoming a better teacher. In particular, this training
   is aimed at those who want to become <a href="{{ site.swc_site }}">Software Carpentry</a>,
-  <a href="{{ site.lc_site }}">Library Carpentry</a>, and <a href="{{ site.dc_site }}">Data Carpentry</a>
+  <a href="{{ site.lc_site }}">Library Carpentry</a>, and <a href="{{ site.https://laughingsquid.com/baby-goats-line-up-to-get-hug-from-human/dc_site }}">Data Carpentry</a>
```

Comment:
This is a strange and delightful addition. Did you mean to add this?

Reply:
Oh that is spectacular. Inadvertent copy + paste bringing joy to the world. I'll update the PR.